### ISSUE TITLE
Support Bitpay as a payment option

### ISSUE THREAD

**Issue by btcdrak** [odoo/odoo#8446]

**Comment by vonpupp** [comment]
I second this also. Bitpay seems to be the way to go.

**Comment by rvalyi** [comment]
not a bug, can it be closed?
cc @Yenthe666

**Comment by Yenthe666** [comment]
@rvalyi,

Usually wishlist items are kept open, no matter how old they are and if it is a bug or not. I'll leave this open for now anyways as it is a good suggestion. 😉

**Comment by Yenthe666** [comment]
@pedrobaeza if we follow that logic we can close off a ton of issues honestly. There are loads of wishlist items open.

**Comment by nhomar** [comment]
@Yenthe666 I feel the labl wishlist is enough to filter them, but manage those internally in some moment in the future, I am agree to close them... But let's see what @mart-e says.

**Comment by Yenthe666** [comment]
Yes. I think we might want to change our policy. I'll send out an e-mail to all of you to discuss this.
